src/Maps/components/LeafletMap.ts

```diff
@@ -0,0 +1,192 @@
+import { Component, createElement } from "react";
+import {
+    FeatureGroup,
+    LatLngLiteral,
+    LeafletEvent,
+    Map,
+    Marker,
+    icon,
+    tileLayer
+} from "leaflet";
+import * as classNames from "classnames";
+
+import { Container, MapUtils } from "../utils/namespace";
+import { Shared } from "../utils/sharedConfigs";
+import { Alert } from "../../components/Alert";
+import MapProps = Container.MapProps;
+import DataSourceLocationProps = Container.DataSourceLocationProps;
+import Location = Container.Location;
+import customUrls = MapUtils.customUrls;
+import mapAttr = MapUtils.mapAttr;
+import SharedProps = MapUtils.SharedProps;
+
+export type LeafletMapProps = {
+    onClickMarker?: (event: LeafletEvent, locationAttr: DataSourceLocationProps) => void
+} & SharedProps & MapProps;
+
+export interface LeafletMapState {
+    center: LatLngLiteral;
+    alertMessage?: string;
+}
+
+export class LeafletMap extends Component<LeafletMapProps, LeafletMapState> {
+    private leafletNode?: HTMLDivElement;
+    private defaultCenterLocation: LatLngLiteral = { lat: 51.9107963, lng: 4.4789878 };
```

Comment:
What do these coordinates point to? Mendix office? Aha, what if it gets moved? You have to come and recode this... perhaps use an address.

Reply:
Widget currently doesn't support addresses